core/drivers/pmd.cc

```diff
@@ -255,17 +282,14 @@ CommandResponse PMDPort::Init(const bess::pb::PMDPortArg &arg) {
     eth_rxconf.rx_drop_en = 1;
   }
 
-  eth_txconf = dev_info.default_txconf;
-  eth_txconf.txq_flags = ETH_TXQ_FLAGS_NOVLANOFFL |
```

Comment:
Don't want to turn off these offloads anymore?

Reply:
I think these are not support anymore:
https://mails.dpdk.org/archives/dev/2018-July/106459.html
https://mails.dpdk.org/archives/dev/2018-June/103883.html

and IIRC offloading is now by default disabled.